packages/addons/service/touchscreen/package.mk

```diff
@@ -20,7 +20,7 @@ PKG_NAME="touchscreen"
 PKG_VERSION="1.0"
 PKG_REV="101"
 PKG_ARCH="any"
-PKG_ADDON_PROJECTS="Generic RPi RPi2"
+PKG_ADDON_PROJECTS="Generic RPi RPi2 S905"
```

Comment:
I'd suggest "Amlogic" instead of "S905". So we can have support for all devices under the Amlogic Project.

Reply:
I can do that but in this case it still doesn't build
`*** ERROR: service.touchscreen: 'S905' not supported ***`

Reply:
+1 this was the idea behind the unification to reduce tidbits like this

Reply:
@vpeter4: Yeah that issue always hits us in the face.

Reply:
Maybe @MilhouseVH can fix the code in scripts/create_addon

Reply:
Ah... that's because $DEVICE takes precedence over $PROJECT when validating the addon. 

What about changing  https://github.com/LibreELEC/LibreELEC.tv/blob/master/scripts/create_addon#L40 to the following, which should allow PROJECT (ie. `Amlogic`) to be validated as well as DEVICE:
```
  if ! listcontains "$PKG_ADDON_PROJECTS" "${DEVICE:-$PROJECT}" && 
     ! listcontains "$PKG_ADDON_PROJECTS" "${$PROJECT}" && 
     ! listcontains "$PKG_ADDON_PROJECTS" "any"; then
    not_supported
  fi
```
?

With the above you could still have `PKG_ADDON_PROJECTS="Amlogic !S912` which would allow the addon to be created for all Amlogic projects but not S912.

The only fly in the ointment is that an addon which is valid for the RPi device but not RPi2 device will now become valid for RPi2 because the validation will interpret RPi as the project not the device. Fortunately we don't have any packages configured in this way, so it's not currently an issue. If it becomes an issue we'll need to sort something out. Note for the future - never name a device with the same name as a project! :)

If the above works (not able to test it right now) you might as well add it to this PR.

Reply:
^^^ Tested, seems to work as expected - please add if it now does what you need.